concent_api/requirements.lock

```diff
@@ -83,4 +90,5 @@ toolz==0.9.0
 typed-ast==1.1.0
 urllib3==1.23
 vine==1.1.4
-web3==3.16.4
+web3==4.2.1
+websockets==5.0.1
```

Comment:
Why does SCI depend on `websockets`?

Reply:
`websockets` depends on `web3` and is installing during it's installation. Look: 
`Collecting websockets>=4.0.1 (from web3==4.2.1)`

Reply:
Weird but OK.